packages/dds/tree/src/simple-tree/proxyBinding.ts

```diff
@@ -93,8 +96,12 @@ export function getFlexNode(proxy: TreeNode, allowFreed = false): FlexTreeNode {
 		const flexNode = anchorNode.slots.get(flexTreeSlot);
 		if (flexNode !== undefined) {
 			return flexNode; // If it does have a flex node, return it...
-		} // ...otherwise, the flex node must be created by walking downwards from an existing ancestor
-		const newFlexNode = demandSpine(anchorNode);
+		} // ...otherwise, the flex node must be created
+		const context = anchorNode.anchorSet.slots.get(ContextSlot) ?? fail("missing context");
+		const cursor = context.forest.allocateCursor();
+		context.forest.moveCursorToPath(anchorNode, cursor);
+		const newFlexNode = makeTree(context, cursor);
```

Comment:
This is awesome! We need to update a few lines in ProxyBinding.md to explain the new behavior.

Reply:
Done.